torchtext/data/datasets_utils.py

```diff
@@ -318,3 +319,26 @@ def pos(self):
 
     def __str__(self):
         return self.description
+
+
+@functional_datapipe("read_squad")
```

Comment:
I guess we could remove this once #1513  lands right?

Reply:
That's right. These are the same commits so I'll just rebase once squad1 lands.